src/components/common/layout/content.scss

```diff
@@ -1,3 +1,8 @@
 .qld-content {
-    padding: 4rem 0 4.5rem 0;
-}
\ No newline at end of file
+    padding: 4rem 1.25rem 4.5rem 1.25rem;
+
+    >.container {
+        // overwriting bootstrap's default 1.5rem gutter value
+        --qld-gutter-x: 2.5rem;
```

Comment:
this should be changed to (--#{$prefix} instead of --qld 
so that prefix overrides are happy.